plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -675,6 +675,15 @@ private static Map<String, String> getColumnNameToStatisticsName(JdbcTableHandle
         return columnNameToStatisticsName;
     }
 
+    @Override
+    protected String escapeObjectNameForMetadataQuery(String name, String escape)
+    {
+        name = super.escapeObjectNameForMetadataQuery(name, escape);
```

Comment:
the call to `super` is correct only because you know what exactly the implementation is doing.
since you know this, you can as well inline

i'd inline here.

Reply:
Under inline you mean:
   `  return super.escapeObjectNameForMetadataQuery(name, escape)`
               ` .replace("]", escape + "]")`
               ` .replace("[", escape + "[");`
?

Reply:
i meant replacing `super.escapeObjectNameForMetadataQuery` with the contents of that method.

cc @hashhar

Reply:
BTW is there some SQL Server docs documenting the deviation from JDBC specification?

Reply:
yes, see https://learn.microsoft.com/en-us/sql/t-sql/language-elements/like-transact-sql?redirectedfrom=MSDN&view=sql-server-ver16#using-wildcard-characters-as-literals and https://learn.microsoft.com/en-us/sql/t-sql/language-elements/like-transact-sql?redirectedfrom=MSDN&view=sql-server-ver16#arguments